pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -53,7 +53,7 @@
 
 , passthru ? {}
 
-, doCheck ? false
+, doCheck ? true
```

Comment:
Isn't this a bit violent? The rest of the PR appears harmless, but wouldn't this set `doCheck` default to `true` regardless of `config.doCheckByDefault`?

Reply:
Fair enough. Will fix.